### DISCUSSION TITLE
Disabling Number column and indent guide by default

### DISCUSSION THREAD

**Discussion by RizaHKhan** [LazyVim/LazyVim#4209] [General]
Is there a way to disable number column and indent guide so start up looks like this:

![image](https://github.com/user-attachments/assets/e42659f0-0a7c-4cca-9daf-434c3babf513)

I tried 

`opt.number = false` but that doesn't seem to work, and I couldn't find anything for indent guides.

I want to be able to toggle them on, but want to off by default.

**Comment by dpetka2001** [comment]
In `options.lua` set
```lua
vim.opt.number = false
vim.opt.relativenumber = false
```
and then for indents create a file `~/.config/nvim/lua/plugins/indent.lua` (or whatever name you like) with contents
```lua
return {
  {
    "lukas-reineke/indent-blankline.nvim",
    opts = {
      enabled = false,
    },
  },
}
```

**Reply by vjanandr** [reply]
the suggestion to remove indent lines doesnt seem to work..
  {
    "lukas-reineke/indent-blankline.nvim",
    opts = {
      enabled = true,
    },
  }
I still see the indent lines..

**Reply by dpetka2001** [reply]
```lua
return {
	{
		"folke/snacks.nvim",
		opts = {
			indent = {
				enabled = false,
			},
		},
	},
}
```
should do it.

**Reply by vjanandr** [reply]
this works thank you..
